react/week2/todo-list-app/src/Components/Mylist.json

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
I would name it data.json and put it outside of Components folder since it is not component

Reply:
Thanks for your time and valuable comment